templates/unstable-vite-cloudflare/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "scripts": {
+    "dev": "remix vite:dev",
+    "build": "remix vite:build",
+    "start": "wrangler pages dev ./build/client --kv MY_KV",
+    "deploy": "wrangler pages deploy ./build/client",
+    "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@remix-run/cloudflare": "*",
+    "@remix-run/cloudflare-pages": "*",
+    "@remix-run/react": "*",
```

Comment:
Currently Remix includes `@remix-run/node` in `optimizeDeps` but this template doesn't have direct dependency on it. This might cause Vite's warning for non-flat node_modules like pnpm where `@remix-run/node` is not visible from root project. 

I have a similar case where I use `@remix-run/server-runtime` directly without `@remix-run/node` and I've seen this warning on startup:

```
Failed to resolve dependency: @remix-run/node, present in 'optimizeDeps.include'
```

I'm not sure what's the best way to avoid this within Remix, but for now, what I did was to exclude manually by custom plugin `config` hook https://github.com/hi-ogawa/demo-remix-unstable-vite-cloudflare-workers/blob/2bdd6a4e15dc99584aa23897a5c1a40d7b185f6e/vite.config.ts#L15-L17